server/fleet/app.go

```diff
@@ -500,6 +502,12 @@ func (c *AppConfig) Copy() *AppConfig {
 		clone.Scripts = optjson.SetSlice[string](scripts)
 	}
 
+	if c.MDM.WindowsSettings.CustomSettings.Set {
+		windowsSettings := make([]string, len(c.MDM.WindowsSettings.CustomSettings.Value))
+		copy(windowsSettings, c.MDM.WindowsSettings.CustomSettings.Value)
+		clone.MDM.WindowsSettings.CustomSettings = optjson.SetSlice[string](windowsSettings)
```

Comment:
Same here, I would hope (but am not sure at all, never tried it) that it'd work without `[string]`?